install.sh

```diff
@@ -90,7 +112,8 @@ function install_packages {
     install_yum
 }
 
-if [ -n "${GDB}" ]; then
+
+if [ "${BACKEND_GDB}" -eq 1 ]; then
```

Comment:
I swear I made comments on this previously. It looks to me like this will stop the auto-detection of debuggers since you're only checking the "backend" flag and not the detected `$GDB` and `$LLDB`. Did you test the auto-detection?